### ISSUE TITLE
Pylint thinks dataclasses isn't a standard import

### ISSUE THREAD

**Issue by chingc** [PyCQA/pylint#2261]
### Steps to reproduce

Given:

```python
import dataclasses
import os
```

Pylint will indicate:

```
C0411: standard import "import os" should be placed before "import dataclasses" (wrong-import-order)
```

### pylint --version

```
pylint 2.0.0.dev2
astroid 2.0.0.dev4
Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
```

**Comment by PCManticore** [comment]
Do you have the third party `dataclasses` installed by any chance? I just tried with a fresh installation of 3.7 and it worked without problems.

**Comment by lbenezriravin** [comment]
I'm seeing the same issue:
```
(testy) @lbenezriravin:~ $ cat tester.py 
import dataclasses
import os
(testy) @lbenezriravin:~ $ pylint tester.py 
************* Module tester
tester.py:1:0: C0111: Missing module docstring (missing-docstring)
tester.py:1:0: W0611: Unused import dataclasses (unused-import)
tester.py:2:0: W0611: Unused import os (unused-import)
tester.py:2:0: C0411: standard import "import os" should be placed before "import dataclasses" (wrong-import-order)

----------------------------------------------------------------------
Your code has been rated at -10.00/10 (previous run: -10.00/10, +0.00)

(testy) @lbenezriravin:~ $ pylint --version
pylint 2.0.0.dev2
astroid 2.0.0.dev4
Python 3.7.0 (default, Jun 28 2018, 08:59:52) 
[GCC 7.3.1 20180303 (Red Hat 7.3.1-5)]
(testy) @lbenezriravin:~ $ 
```

**Comment by brycepg** [comment]
Created a PR in isort which should eventually fix this issue: timothycrosley/isort#719

**Comment by PCManticore** [comment]
Closing as this should be handled by Bryce's PR!
